configs/templates/unified-dev/spack.yaml

```diff
@@ -24,16 +24,11 @@ spack:
     - neptune-env           ^esmf@=8.8.0
     - neptune-python-env    ^esmf@=8.8.0
     - soca-env
-    - ufs-srw-app-env       ^esmf@=8.6.1
-    - ufs-weather-model-env ^esmf@=8.6.1
+    - ufs-srw-app-env       ^esmf@=8.6.1 ^crtm@=3.1.1-build1
+    - ufs-weather-model-env ^esmf@=8.6.1 ^crtm@=3.1.1-build1
 
-    # Various crtm tags (list all to avoid duplicate packages)
-    - crtm@2.4.0.1
```

Comment:
This should not be removed. THere are still partners who use this version of crtm.

Reply:
I'll fix this when I also fix the crtm download issue